components/ws-daemon/pkg/content/initializer.go

```diff
@@ -137,12 +143,15 @@ func RunInitializer(ctx context.Context, destination string, initializer *csapi.
 	msg := msgInitContent{
 		Destination:   "/dst",
 		Initializer:   init,
-		RemoteContent: remoteContent,
```

Comment:
RemoteContent why this was removed ?

Reply:
We replace it with
 		Owner:         opts.Owner,
		Workspace:     opts.Workspace,
		StorageConfig: opts.Config,

This is the info used to create the remoteContent object. We can pass this info instead of the object as it gives more flexibility.